generated/beats/fields.ecs.yml

```diff
@@ -1267,6 +1267,16 @@
         is relevant to Apple *OS only.'
       example: EQHXZ8M8AV
       default_field: false
+    - name: code_signature.thumbprint
+      level: extended
+      type: keyword
+      ignore_above: 1024
```

Comment:
Is there a way to specify fixed width and enforce lowercase?

```
pattern: ^[a-f0-9]{64}$
```

Reply:
It's not possible to enforce this within ECS

Reply:
I saw `pattern` specified for a few other fields. So this is just a hint?
Even as a hint it seems like a useful way of describing expected values.

Reply:
Yes, there's currently no implementation which will enforce that the pattern is correct, so it's really just a hint now. This is something I was looking into separately from this issue already though. I plan to research how to set up enforcement in the near future.